src/components/movies/MovieCard.vue

```diff
@@ -1,17 +1,20 @@
 <template>
     <div class="movie__card">
         <h2 class="movie__title"> {{ movieTitle }}</h2>
-        <p class="movie__duration-time">{{ movieLength }}</p>
-        <img class="movie__poster" :src="moviePoster" />
-        <base-chip class="movie__category">
-            {{ movieCategory }}
-        </base-chip>
+        <div class="movie__details">   
+            <p class="movie__details--duration-time">{{ movieLength }}</p>
+            <img class="movie__details--poster" :src="moviePoster" />
+            <base-chip class="movie__details--category">
```

Comment:
It's great that you're using BEM to name CSS classes, but I have a small suggestion here, as I understand `time`, `poster`, `category` are movie block elements so I propose to change the following classes this way:
`movie__details--duration-time` => `movie__details-duration` (here duration is sufficient)
`movie__details--poster` => `movie__details-poster`
`movie__details--category` => `movie__details-category`

As you know the `--` sign is used to create modifiers classes `Flags on blocks or elements. Use them to change appearance, behavior or state.`. For example if moviePoster content should be red then we can create a modifier class for it like this `movie__details-poster---red` where `---red` changes the appearance of the movie poster.

I know that this fix could looks not so important but it's good to keep clarity in the css code also :) 
Adding a linkg to [the BEM docs](http://getbem.com/naming/)

Reply:
I appreciate that 🙏. I was confused in some cases even if I have read BEM docs how to name classes and I needed some one to emphasize this. Another thing is naming nested elements and I found this [article] (https://scalablecss.com/bem-nesting-grandchild-elements/). Is this approach ok? 🧐

Reply:
I understand I've had the same problem when I started to use BEM, I took a look at this article and I think this approach is ok, it's much clearer and in the case of many classes, you'll still have control over them.